packaging/opensuse-42.2/snapd.spec

```diff
@@ -164,15 +164,15 @@ make %{?_smp_mflags} -C cmd check
 # Install all the go stuff
 %goinstall
 # TODO: instead of removing it move this to a dedicated golang package
-rm -rf %{buildroot}/usr/lib64/go
-rm -rf %{buildroot}/usr/lib/go
+rm -rf %{buildroot}%{_libexecdir}64/go
```

Comment:
Looking at RPM macros I don't see anything relevant. I _think_ it's weird to do it this way and I'd use /usr/lib64/go for this. Alternatively just use `%{_libdir}`